README.md

```diff
@@ -93,6 +93,15 @@ FIRECRAWL_API_KEY=
 ARCADE_API_KEY=
 ```
 
+If you plan to post to LinkedIn as an organization (rather than as yourself), you'll also need to set:
+
+```bash
+# Get the organization ID from the URL of the company page when you're logged in as an admin.
+# For example, if the URL is `https://www.linkedin.com/company/12345678/admin/dashboard/`, the organization ID would be `12345678`.
+POST_TO_LINKEDIN_ORGANIZATION=true
+LINKEDIN_ORGANIZATION_ID=
```

Comment:
With this, I was able to post to LinkedIn as an org. Without `LINKEDIN_ORGANIZATION_ID`, the agent logged an error about this var being missing.